modules/nodes-nodes-aws-autoscaler-LC-and-ASG.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-aws-autoscaler.adoc
+
+[id='nodes-nodes-aws-autoscaler-primed-images-{context}']
+= Creating a primed image for the AWS auto-scaler
+
+You can use Ansible playbooks to automatically create a
+primed image for the auto-scaler to use. You must provide attributes
+from your existing Amazon Web Services (AWS) cluster.
+
+[NOTE]
+====
+If you already have a primed image, you can use it instead of creating a new one.
+====
+
+.Procedure
+
+On the host that you used to create your {product-title} cluster, create a
+primed image:
+
+. Create a new Ansible inventory file on your local host:
++
+----
+[OSEv3:children]
+masters
+nodes
+etcd
+
+[OSEv3:vars]
+ifdef::openshift-enterprise[]
+openshift_deployment_type=openshift-enterprise
+endif::[]
+ifdef::openshift-origin[]
+openshift_deployment_type=origin
+endif::[]
+ansible_ssh_user=ec2-user
+openshift_clusterid=mycluster
+ansible_become=yes
+
+[masters]
+[etcd]
+[nodes]
+----
+
+. Create provisioning file, *_build-ami-provisioning-vars.yaml_*, on your local
+host:
++
+[source,yaml]
+----
+ifdef::openshift-enterprise[]
+openshift_deployment_type: openshift-enterprise
+endif::[]
+ifdef::openshift-origin[]
+openshift_deployment_type: origin
```

Comment:
@mburke5678 - should this be OKD?

Reply:
I will verify. These sections were already split into modules, which I copied and pasted straight into new files. 
Are you suggesting the parameter should be `openshift_deployment_type: okd` or questioning the ifdef? The ifdef appears to set the deployment type properly for OSV vs OKD

Reply:
@mburke5678 I was worried that it should be openshift_deployment_type: okd.

Reply:
@vikram I am not clear. Is the definition for the `ifdef::openshift-origin` changing from `openshift_deployment_type: origin` to `openshift_deployment_type: okd`?
```
ifdef::openshift-origin[]
openshift_deployment_type: okd
```